trusted_os/ctl.go

```diff
@@ -188,9 +188,10 @@ func (ctl *controlInterface) handleLogsRequest(r []byte, l func() []byte) (res [
 		log.Printf("Compressed %d bytes of log messages to %d send", ll, len(ctl.logBuffer))
 	}
 	ret := &api.LogMessagesResponse{}
-	if l := len(ctl.logBuffer); l > maxChunkSize {
+	logChunk := 1024
+	if l := len(ctl.logBuffer); l > logChunk {
 		ret.More = true
-		ret.Payload, ctl.logBuffer = ctl.logBuffer[:maxChunkSize], ctl.logBuffer[maxChunkSize:]
```

Comment:
`maxChunkSize` is not used after this PR, should it stay around?

Reply:
Possibly... I'll leave it for now and have a look around, and send a follow-up to remove.